components/Stats.vue

```diff
@@ -34,35 +95,48 @@ import TimeBarChart from '@/components/TimeBarChart.vue'
 import CasesSummary from '@/components/CasesSummary.vue'
 import Data from '@/data/data.json'
 import formatCountyData from '@/utils/formatCountyData'
+import consolidateAllData from '@/utils/consolidateAllData'
+import DataView from '@/components/DataView.vue'
+import {
+  calculateDeathRate,
+  calculateTotalCases,
+  calculateCaseRate
+} from '@/utils/calculations'
 
 export default {
   components: {
     CasesSummary,
-    TimeBarChart
+    TimeBarChart,
+    DataView
   },
   data() {
+    const currentCounty = 'San Francisco County'
```

Comment:
needs to implement callback to change the currentCounty user selected

Reply:
It was previously working since I was using a v-model on the select input, but it broke since I tried changing the { countyDeathRate } to a calculated variable